BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/BeaconManager.java

```diff
@@ -80,7 +80,11 @@ public static AdvertisingPacket processAdvertisingPacket(String macAddress, Adve
         beacon.addAdvertisingPacket(advertisingPacket);
         //TODO move outside method
         processClosestBeacon(beacon);
-        instance.notifyBeaconUpdateListeners(beacon);
+        try {
+            instance.notifyBeaconUpdateListeners(beacon);
```

Comment:
So, if an exception is thrown, we don't notify the listener, but it is not crucial for our application?